controller/work_item_type_group_blackbox_test.go

```diff
@@ -0,0 +1,62 @@
+package controller_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/app/test"
+	. "github.com/fabric8-services/fabric8-wit/controller"
+	"github.com/fabric8-services/fabric8-wit/gormapplication"
+	"github.com/fabric8-services/fabric8-wit/gormsupport/cleaner"
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/migration"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	testsupport "github.com/fabric8-services/fabric8-wit/test"
+	almtoken "github.com/fabric8-services/fabric8-wit/token"
+	"github.com/goadesign/goa"
```

Comment:
Thanks @hectorj2f for spending time on it.
Please see the comment and the reason for why I added the change plus why this build is failing.
https://github.com/fabric8-services/fabric8-wit/pull/1541#issuecomment-319932126